test_wrappers.py

```diff
@@ -156,6 +170,927 @@ def test_aria2c(run):
     )
 
 
+def test_agat_levels(run):
+    run(
+        "bio/agat",
+        [
+            "snakemake",
+            "--cores",
+            "1",
+            "--use-conda",
+            "-F",
+            "agat_levels.yaml",
+        ],
+    )
+
+
+def test_agat_convert_bed2gff(run):
+    run(
+        "bio/agat",
+        [
+            "snakemake",
+            "--cores",
+            "1",
+            "--use-conda",
+            "-F",
+            "test_agat_convert_bed2gff.gff",
+        ],
+    )
+
+
+def test_agat_converter_embl2gff(run):
+    run(
+        "bio/agat",
+        [
+            "snakemake",
+            "--cores",
+            "1",
+            "--use-conda",
+            "-F",
+            "test_agat_convert_embl2gff.gff",
+        ],
+    )
```

Comment:
Since `agat` adds a lot of tests, and to avoid cluttering the screen too much, maybe we can just call it once with all target files:

```suggestion
def test_agat(run):
    run(
        "bio/agat",
        [
            "snakemake",
            "--cores",
            "1",
            "--use-conda",
            "-F",
            "agat_levels.yaml",
            "test_agat_convert_bed2gff.gff",
            "test_agat_convert_embl2gff.gff",
        ],
    )
```

Reply:
Yes, you're right. I'm currently reducing dataset tests size. I'll join all separate tests into one just after.